project-bom/pom.xml

```diff
@@ -321,6 +327,12 @@
                 <groupId>org.apache.velocity</groupId>
                 <artifactId>velocity</artifactId>
                 <version>1.7</version>
+                <exclusions>
```

Comment:
this scares me.. `lang` and `lang3` have different namespaces.. I don't think `velocity` will let us swap one out for the other.